### PR TITLE
Yd/varatube updates

### PR DESCRIPTION
**Pull request by YauheniDraichykau**

### CODE REVIEW

**Review by faizov — APPROVED** [review]

**Review by ereburg — APPROVED** [review]

File: frontend/packages/ui/src/components/menu-options/menu-options.module.css [hunk 1]
```diff
@@ -12,10 +12,28 @@
...
+.userSvg {
   margin-left: 2px;
 }
 
-.user-svg path {
+.svg path {
   fill: #000000
+}
+
+.disconnectButton.disconnectButton {
```
Comment (nikitayutanov) [inline frontend/packages/ui/src/components/menu-options/menu-options.module.css:23]:
```suggestion
.disconnectButton {
```
is it typo?

File: frontend/packages/ui/src/components/modal/modal.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import { useEffect, useRef, MouseEvent, useCallback } from 'react';
```
Comment (nikitayutanov) [inline frontend/packages/ui/src/components/modal/modal.tsx:1]: `Modal` already exists in a `@gear-js/vara-ui`. We should extend and improve UI components from library, so let's not duplicate them.

File: frontend/packages/ui/src/features/wallet-new/hooks/use-wallet.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { useAccount } from '@gear-js/react-hooks';
+import { useState } from 'react';
+import { WALLET } from '../consts';
+import { WalletId } from '../types';
+
+function useWallet() {
```
Comment (nikitayutanov) [inline frontend/packages/ui/src/features/wallet-new/hooks/use-wallet.ts:6]: Let's reuse `useWallet` hook from original implementation. No need to repeat the logic.

This applies to shared `assets` as well.

File: frontend/packages/ui/src/features/wallet-new/hooks/use-free-account-balance.ts [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import { useAccount, useApi } from '@gear-js/react-hooks';
+import { useEffect, useState } from 'react';
+
+function useFreeAccountBalance() {
```
Comment (nikitayutanov) [inline frontend/packages/ui/src/features/wallet-new/hooks/use-free-account-balance.ts:4]: In the latest `@gear-js/react-hooks` version we have this hook already.

**Review by nikitayutanov — CHANGES_REQUESTED** [review]

File: frontend/packages/ui/src/components/menu-options/menu-options.module.css (hunk 1 above)
Comment (YauheniDraichykau) [reply frontend/packages/ui/src/components/menu-options/menu-options.module.css:23]: it's styles specificity increasing. Styles of button can overlap custom ones sometimes

**Review by nikitayutanov — APPROVED** [review]

**Pull request merged**